keyboards/rorschach/keymaps/insertsnideremarks/config.h

```diff
@@ -0,0 +1,30 @@
+
+#ifndef CONFIG_USER_H
```

Comment:
If you could remove the ifndef and define blocks (as well as the endif at the end) and replace it with `#pragma once`?